test/cli/unusedFunction/4.c

```diff
@@ -0,0 +1,7 @@
+static void f4_0() {}
```

Comment:
This was a gap in the test coverage which was uncovered by me not properly propagating all the data in an earlier version of these changes.